client/src/components/Header.js

```diff
@@ -7,10 +7,21 @@ import Select from '@material-ui/core/Select';
 import MenuItem from '@material-ui/core/MenuItem';
 import Button from '@material-ui/core/Button';
 import useStyles from './HeaderStyles';
+import Login from './login/Login';
 
 export default function Header() {
   const classes = useStyles();
 
+  const [show, showModal] = React.useState(false);
+  const handleOpen = () => {
+    showModal(true);
+    document.body.style.overflow = 'hidden';
```

Comment:
it's better not to use inline styles; 
here you can use `if` statement to hide elements correctly